LiskHQ/lisk-desktop
# Display graphical statistics of blocks - Closes #2458

### What issue have I solved?

Resolves #2458

### How have I implemented/fixed it?
Added box with statistics of blocks for 10, 50 and 100 last blocks using `Chart` component.

- [x] Style label on x axis on bar chart.
- [x] Fix tooltips.
- [x] Fix tests - so far the `Chart` component is not being identified on the unit tests.
- [x] Make `t` work on options objects.

### How has this been tested?
1. Run the app.
2. Login to the app using Mainnet or Testnet (Monitor data ONLY works for these networks)
3. Go to Monitor Page
4. Check the Blocks Tab
5. The 2  charts should render properly


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
